planning/behavior_path_planner/autoware_behavior_path_static_obstacle_avoidance_module/src/scene.cpp

```diff
@@ -570,6 +570,24 @@ void StaticObstacleAvoidanceModule::fillEgoStatus(
     return;
   }
 
+  auto registered_sl_force_deactivated =
+    [&](const std::string & direction, const RegisteredShiftLineArray shift_line_array) {
+      return std::any_of(
+        shift_line_array.begin(), shift_line_array.end(), [&](const auto & shift_line) {
+          return rtc_interface_ptr_map_.at(direction)->isForceDeactivated(shift_line.uuid);
+        });
+    };
+
+  auto is_force_deactivated = registered_sl_force_deactivated("left", left_shift_array_) ||
```

Comment:
```suggestion
  const auto is_force_deactivated = registered_sl_force_deactivated("left", left_shift_array_) ||
```

Reply:
Fixed in 22fa38d5b09c711d4e91948907a4f7af5bcbc193